src/local-editor-service.ts

```diff
@@ -140,6 +139,24 @@ export class LocalEditorService extends EditorService {
     };
   }
 
+  _generateAutoCompletionForElement(e: Element): string {
+    let autocompletion = `<${e.tagName}`;
+    let tabindex = 1;
+    if (e.attributes.length) {
+      autocompletion += ` $${tabindex++}`;
```

Comment:
To not make this a breaking change, we could both add the `expandTo` and `expandToSnippet` just like the solution with `descriptionMarkdown`. Then `expandTo` is a "dumb" implementation and `expandToSnippet` is a smart implementation